src/test/java/io/strimzi/kafka/bridge/amqp/AmqpBridgeTest.java

```diff
@@ -843,4 +845,13 @@ public void noPartitionsAvailable(TestContext context) {
 			}
 		});
 	}
+
+	@Test
+	public void ConvertedMessageWithNullKey(TestContext context) {
+		byte[] value = {Byte.MIN_VALUE,Byte.MAX_VALUE};
+		ConsumerRecord<String,byte[]> record = new ConsumerRecord<String, byte[]>("mytopic",0,0,null,value);
+		AmqpDefaultMessageConverter messageConverter = new AmqpDefaultMessageConverter();
+		Message message = messageConverter.toMessage("0",record);
+		context.assertNull(message.getMessageAnnotations().getValue().get(Symbol.valueOf(AmqpBridge.AMQP_KEY_ANNOTATION)));
+	}
```

Comment:
Thanks for the added tests ! A couple of comments here.
Out of curiosity ... why that kind of payload ? :-)
Then you should need in the same test method, the check for `null` key even for the other two converters (raw and json).

Reply:
I have added test for jsonMessageConverter too and also corrected the payload. :D 
but the rawMessageConverter throws org.apache.qpid.proton.codec.DecodeException: Unknown constructor for same payload. i couldn't find the reason.